ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/ScenarioLoadingITBase.scala

```diff
@@ -129,17 +129,20 @@ abstract class ScenarioLoadingITBase
           lookForContract(events, templateIds.dummyFactory)
           lookForContract(events, templateIds.dummyContractFactory)
 
-          resp.last should equal(GetActiveContractsResponse("8", "", Seq.empty, None))
+          val GetActiveContractsResponse(offset, workflowId, activeContracts, _) = resp.last
+          offset should not be empty
+          workflowId shouldBe empty
+          activeContracts shouldBe empty
```

Comment:
Nice. I actually tried making the same changes to `JdbcLedgerDao` when I saw the level of duplication there, but couldn't figure out what to do with this resulting test failure.